src/ucs/async/eventfd.c

```diff
@@ -0,0 +1,88 @@
+/**
+* Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "eventfd.h"
+
+#include <ucs/debug/log.h>
+#include <ucs/sys/sys.h>
+
+
+typedef ssize_t (*ucs_async_eventfd_cb_t)(int fd, void *buf, size_t count);
+
+
+static inline ucs_status_t
+ucs_async_eventfd_common_io(int fd, int blocking, ucs_async_eventfd_cb_t cb)
+{
+    uint64_t dummy = 1;
```

Comment:
char dummy = 0;

Reply:
@yosefe Making this change seems to cause worker to not add stream callbacks through cuda_iface. Not sure why. Also, all of this code is borrowed from UCP worker fd read/write handling code. Why is `uint64_t dummy = 1` used [here](https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_worker.c#L410) instead of say `char dummy = 0`?

Reply:
ok